packages/cli/src/oclif/commands/legacy.js

```diff
@@ -0,0 +1,67 @@
+const BaseCommand = require('../ZapierBaseCommand');
+const { Args, Flags } = require('@oclif/core');
+const { buildFlags } = require('../buildFlags');
+const colors = require('colors/safe');
+
+const { callAPI } = require('../../utils/api');
+
+class LegacyCommand extends BaseCommand {
+  async perform() {
+    const app = await this.getWritableApp();
+    const { version } = this.args;
+
+    this.log(
+      `${colors.yellow('Warning: .')}\n` +
```

Comment:
Is there an extra dot or is that on purpose for formatting?
```suggestion
      `${colors.yellow('Warning:')}\n` +
```

Reply:
Whoops - it was a typo. But also, let's omit the warning here (warning mentioned using zapier migrate!)

Reply:
But it led me to add another bit about migrating to legacy (and deprecated versions) [here](https://gitlab.com/zapier/public-api/public-api-docs/-/merge_requests/56/diffs?commit_id=d3fa78f394e426102c343b1bc64f369e5e7f0d2a)